src/core_init_atmosphere/mpas_init_atm_gwd.F

```diff
@@ -591,20 +782,23 @@ subroutine get_box(lat, lon, dx)
          do while (ii > topo_x)
             ii = ii - topo_x
          end do
- 
-         box(i,j) = topo(ii,jj)
-         box_landuse(i,j) = landuse(ii,jj)
+         l = l + 1
+      
+         ! Obtain tile for given box pixel from the linked list of tiles (tilesHead),
+         ! which would involve reading in the data from disk if said tile is not already in memory
+         thisTile => get_tile_from_box_point(tilesHead, ii, jj, path, sub_path)
+
+         tile_offset = (ii - thisTile%tile_start_x) + (jj - thisTile%tile_start_y) * tile_x + 1
+
+         box(i,j) = thisTile%topo_array(tile_offset)
+         box_landuse(i,j) = thisTile%land_array(tile_offset)
          sg_lat = (start_lat + (real(jj-1,RKIND) + 0.5) / pts_per_degree) / rad2deg  ! Add 0.5 for cell center
          dxm(i,j) = sg_delta * cos(sg_lat)
          box_mean = box_mean + box(i,j)
-
+ 
```

Comment:
It's probably better to keep the blank line as it was.

Reply:
Done.